docs/developers-guide/driver-changelog.md

```diff
@@ -6,16 +6,22 @@ title: Driver interface changelog
 
 ## Metabase 0.47.0
 
-- The multimethod `metabase.driver/syncable-schemas` has been added. This method is used to list schemas to upload
-  CSVs to, and it should include all schemas that are able to be synced to. Currently it only needs to be implemented
-  if the database has schema, and the database supports the `uploads` feature.
-
 - A new driver feature has been added: `:schemas`. This feature signals whether the database organizes tables in
-  schemas (also known as namespaces) or not. Most databases have schemas so this feature is supported on by default.
+  schemas (also known as namespaces) or not. Most databases have schemas so this feature is on by default.
   An implemention of the multimethod `metabase.driver/database-supports?` for `:schemas` is required only if the
   database doesn't store tables in schemas.
 
-- The multimethod `metabase.driver/supports?` has been deprecated in favor of `metabase.driver/database-supports?`. The existing default implementation of `database-supports?` currently calls `supports?`, but it will be removed in 0.55.0.
+- Another driver feature has been added: `uploads`. The `uploads` feature signals whether the database supports 
+  uploading CSV files to tables in the database. To support the uploads feature, implement the following new 
+  multimethods: `metabase.driver/create-table!` (creates a table), `metabase.driver/drop-table!` (drops
+  a table), and `metabase.driver/insert-into!` (inserts values into a table).
+
+- The multimethod `metabase.driver/syncable-schemas` has been added. This method is used to list schemas to upload
+  CSVs to, and it should include all schemas that are able to be synced to. Currently it only needs to be implemented
```

Comment:
I'm not a native English speaker so feel free to ignore.
```suggestion
  CSVs to, and it should include all schemas that can be synced. Currently it only needs to be implemented
```

Reply:
I am, and I agree this edit is clearer.